Description: Upstream changes introduced in version 1.8-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 yubikey-ksm (1.8-1) unstable; urgency=low
 .
   * Initial package.
 .
 The person named in the Author field signed this changelog entry.
Author: Simon Josefsson <simon@josefsson.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ yubikey-ksm-1.8/files/etc/logrotate.d/yubikey-ksm
@@ -0,0 +1,9 @@
+/var/log/yubikey-ksm.log {
+	weekly
+	missingok
+	rotate 9999
+	notifempty
+	postrotate
+		invoke-rc.d rsyslog reload > /dev/null
+	endscript
+}
--- /dev/null
+++ yubikey-ksm-1.8/files/etc/php5/conf.d/ykksm.ini
@@ -0,0 +1 @@
+include_path = "/etc/ykksm:/usr/share/yubikey-ksm"
--- /dev/null
+++ yubikey-ksm-1.8/files/etc/rsyslog.d/yubikey-ksm.conf
@@ -0,0 +1 @@
+local0.* -/var/log/yubikey-ksm.log
--- /dev/null
+++ yubikey-ksm-1.8/files/etc/apache2/sites-available/yubikey-ksm-ssl
@@ -0,0 +1,40 @@
+<IfModule mod_ssl.c>
+
+<VirtualHost *:443>
+        ServerName api.example.com
+        ServerAdmin support@example.com
+
+        SSLEngine on
+        SSLCertificateFile /etc/ssl/certs/ssl-cert-snakeoil.pem
+	#SSLCertificateChainFile /etc/apache2/ssl.crt/server-ca.crt
+        SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
+
+        DocumentRoot /var/www/
+        <Directory />
+                Options None
+                AllowOverride None
+                Order allow,deny
+                Deny from all
+        </Directory>
+
+	Alias /wsapi/2.0/verify		/usr/share/yubikey-val-server-php/2.0/ykval-verify.php
+	Alias /wsapi/2.0/sync		/usr/share/yubikey-val-server-php/2.0/ykval-sync.php
+
+        <Directory /usr/share/yubikey-val-server-php/>
+                Options None
+                AllowOverride None
+                Order allow,deny
+                Allow from all
+		<IfModule mod_php5.c>
+			  php_value include_path ".:/etc/ykval:/usr/share/yubikey-val-server-php/2.0"
+		</IfModule>
+        </Directory>
+
+        ErrorLog /var/log/apache2/yubikey-val-server-php-ssl-error.log
+        LogLevel warn
+
+        CustomLog /var/log/apache2/yubikey-val-server-php-ssl-access.log "%h %l %u %t \"%r\" %>s %b %D \"%{Referer}i\" \"%{User-Agent}i\""
+        ServerSignature On
+</VirtualHost>
+
+</IfModule>
--- /dev/null
+++ yubikey-ksm-1.8/files/etc/apache2/sites-available/yubikey-ksm
@@ -0,0 +1,30 @@
+<VirtualHost *:80>
+        ServerName api.example.com
+        ServerAdmin support@example.com
+
+        DocumentRoot /var/www/
+        <Directory />
+                Options None
+                AllowOverride None
+                Order allow,deny
+                Deny from all
+        </Directory>
+
+	Alias /wsapi/decrypt		/usr/share/yubikey-ksm/ykksm-decrypt.php
+
+        <Directory /usr/share/yubikey-ksm/>
+                Options None
+                AllowOverride None
+                Order allow,deny
+                Allow from all
+		<IfModule mod_php5.c>
+			  php_value include_path ".:/etc/ykksm:/usr/share/yubikey-ksm"
+		</IfModule>
+        </Directory>
+
+        ErrorLog /var/log/apache2/yubikey-ksm-error.log
+        LogLevel warn
+
+        CustomLog /var/log/apache2/yubikey-ksm-access.log "%h %l %u %t \"%r\" %>s %b %D \"%{Referer}i\" \"%{User-Agent}i\""
+        ServerSignature On
+</VirtualHost>
